wolfcrypt/src/asn.c

```diff
@@ -32696,6 +32696,12 @@ int wc_EccPrivateKeyToDer(ecc_key* key, byte* output, word32 inLen)
     return wc_BuildEccKeyDer(key, output, &inLen, 0, 1);
 }
 
+/* Write only private ecc key to DER format,
+ * length on success else < 0 */
+int wc_EccPrivateKeyToDerNoCurve(ecc_key* key, byte* output, word32 inLen)
```

Comment:
Why a new ECC API? Fairly certain you can use `wc_ecc_export_private_only` for this. Otherwise just remove the static on `wc_BuildEccKeyDer` and make `WOLFSSL_LOCAL` in ecc.h and use it from the aria-* code.

Reply:
I've picked the second option. Didn't want to make a static function non-static just for some ARIA code. Figured there was a reason why it was static.